code/modules/atmospherics/machinery/components/binary_devices/pressure_valve.dm

```diff
@@ -154,6 +164,15 @@
 		to_chat(user, "<span class='warning'>You cannot unwrench [src], turn it off first!</span>")
 		return FALSE
 
+/obj/machinery/atmospherics/components/binary/pressure_valve/multitool_act(mob/living/user, obj/item/multitool/I)
+	. = ..()
+	if (istype(I))
+		inverted = !inverted
+		if(inverted)
+			to_chat(user, "<span class='notice'>You set [src]'s sensors to release gases when the pressure is higher than the set one.</span>")
+		else
+			to_chat(user, "<span class='notice'>You set [src]'s sensors to the default settings.</span>")
+	return TRUE
```

Comment:
Does inverted actually change the pressure requirements? Was this tested?

Reply:
F i forgot to add the change to the actual process